honeybee_schema/project_info.py

```diff
@@ -0,0 +1,91 @@
+"""Schema for project information."""
+from pydantic import BaseModel, Field, constr, AnyUrl
+from typing import List
+
+from .energy.simulation import EfficiencyStandards, ClimateZones, BuildingTypes
+
+
+class Location(BaseModel):
+    """A Ladybug Location."""
+
+    type: constr(regex='^Location$') = 'Location'
+
+    city: str = Field(
+        '-',
+        description='Name of the city as a string.'
+    )
+
+    latitude: float = Field(
+        0,
+        description='Location latitude between -90 and 90 (Default: 0).'
+    )
+
+    longitude: float = Field(
+        0,
+        description='Location longitude between -180 (west) and 180 (east) (Default: 0).'
+    )
+
+    time_zone: float = Field(
+        None,
+        description='Time zone between -12 hours (west) and +14 hours (east). '
+        'If None, the time zone will be an estimated integer value derived from '
+        'the longitude in accordance with solar time.'
+    )
+
+    elevation: float = Field(
+        0,
+        description='A number for elevation of the location in meters. (Default: 0).'
+    )
+
+    station_id: str = Field(
+        None,
```

Comment:
Also not permissible in .NET. Any empty string would be ok.

Reply:
Actually, string can be "None" (null) in .NET